compiler/src/iree/compiler/Codegen/Dialect/VectorExt/IR/VectorExtOps.cpp

```diff
@@ -655,6 +655,14 @@ void TransferGatherOp::getCanonicalizationPatterns(RewritePatternSet &results,
       ctx);
 }
 
+// MaskableOpInterface methods.
+
+/// Returns the mask type expected by this operation. Mostly used for
+/// verification purposes. It requires the operation to be vectorized."
```

Comment:
I'd drop the comment because it is duplicating the comment from interface method. It can be outdated if people update the doc. People who read the code and want to understand what it is, eventually need to go to the interface file, so I think it is okay to drop the dup comment.

https://github.com/llvm/llvm-project/blob/ac7e3910350aed59495883d4193275106047645f/mlir/include/mlir/Dialect/Vector/Interfaces/MaskableOpInterface.td#L59-L61